packages/editor-ui/src/components/PromptMfaCodeModal/PromptMfaCodeModal.vue

```diff
@@ -6,6 +6,7 @@ import { useI18n } from '@/composables/useI18n';
 import { promptMfaCodeBus } from '@/event-bus';
 import type { IFormInputs } from '@/Interface';
 import { createFormEventBus } from 'n8n-design-system';
+import { validate as uuidValidate } from 'uuid';
```

Comment:
To not speak like Yoda ;) 

```suggestion
import { validate as validateUuid } from 'uuid';
```

Reply:
🤣